index.js

```diff
@@ -2,21 +2,21 @@
 const ipRegex = require('ip-regex');
 const tlds = require('tlds');
 
-module.exports = options => {
+module.exports = function(options) {
 	options = {
 		strict: true,
 		...options
```

Comment:
the spread operate breaks in IE11 as well though so that needs to be swapped out